MonoGame.Framework/Graphics/GraphicsCapabilities.DirectX.cs

```diff
@@ -25,11 +25,13 @@ private void PlatformInitialize(GraphicsDevice device)
             SupportsTextureArrays = device.GraphicsProfile == GraphicsProfile.HiDef;
             SupportsDepthClamp = device.GraphicsProfile == GraphicsProfile.HiDef;
             SupportsVertexTextures = device.GraphicsProfile == GraphicsProfile.HiDef;
+
+            _maxMultiSampleCount = GetMaxMultiSampleCount(device);
         }
 
         private void PlatformInitializeAfterResources(GraphicsDevice device)
         {
-            _maxMultiSampleCount = GetMaxMultiSampleCount(device);
+            
```

Comment:
Only DX platform used this method. If you can keep this change we can remove it. It was a bit of a hack anyway.